### PR TITLE
test: Make HR test cleanup `void` instead of `async Task`

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by ajpinedam — APPROVED** [review]

File: src/Uno.UI.RuntimeTests/Tests/HotReload/Given_HotReloadWorkspace.cs [hunk 1]
```diff
@@ -97,7 +97,7 @@ public async Task Initialize()
 	}
 
 	[TestCleanup]
-	public async Task TestCleanupWrapper()
+	public void TestCleanupWrapper()
```
Comment (jeromelaban) [inline src/Uno.UI.RuntimeTests/Tests/HotReload/Given_HotReloadWorkspace.cs:100]: @Youssef1313 The original intent of this change was to ensure that kill was not a fire-and-forget. What's behavior with Task?

Comment (Youssef1313) [reply src/Uno.UI.RuntimeTests/Tests/HotReload/Given_HotReloadWorkspace.cs:100]: In that case, I think we will need to call `WaitForExit`?

Comment (jeromelaban) [reply src/Uno.UI.RuntimeTests/Tests/HotReload/Given_HotReloadWorkspace.cs:100]: Maybe, just to be sure. What was the issue originally?

Comment (Youssef1313) [reply src/Uno.UI.RuntimeTests/Tests/HotReload/Given_HotReloadWorkspace.cs:100]: When tests fail and are re-tried, there is a race where the re-try *may* fail saying file is used by another process. Hoping it doesn't happen now that we wait for the process to exit.

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**